src/Serialization/Upgrades/1.0.0.jl

```diff
@@ -0,0 +1,20 @@
+################################################################################
+# Upgrade Summary
+# 
+# - `ZZLatWithIsom`: we now only serialized the ambient `QuadSpaceWithIsom` and
+# a basis matrix of the underlying lattice; the upgrade recovers the basis matrix
+# in the old tree.
+
+push!(upgrade_scripts_set, UpgradeScript(
+  v"1.0.0",
```

Comment:
technically you should put the version of oscar that this will be released with, 
not exactly sure what this will be, but probably 1.1.0?

Reply:
Ah yes indeed, I misunderstood it and put the current version. I will update this, thank you!